packages/shared/src/modules/activities/libs/types/activity-response-dto.type.ts

```diff
@@ -6,6 +6,7 @@ import { type ActivityPayloadMap } from "./activity-payload-map.type.js";
 type ActivityResponseDto<T extends ValueOf<typeof ActivityType>> = {
 	commentCount: number;
 	id: number;
+	isLikedByUser: boolean | null;
```

Comment:
```suggestion
	isLikedByUser: boolean;
```
should be `boolean`. add `?? false` in cases where we need to handle it